src/Login/Login.tsx

```diff
@@ -21,7 +25,7 @@ class Login extends React.Component<any, LoginState> {
     this.retrieveUser = this.retrieveUser.bind(this);
   }
 
-  retrieveUser(e: any) {
+  retrieveUser(e: any): void {
```

Comment:
you should be able to type this better. React.ReactEvent or something. Look it up